Source-Codes/IERSystem/IERSystem/Areas/Administrator/Views/FormKQ/Edit.cshtml

```diff
@@ -95,158 +90,199 @@
+                <label class="control-label col-md-2 col-sm-2 col-xs-12" for="first-name">
+                    Đơn vị
+                </label>
+                <div class="col-md-4 col-sm-4 col-xs-12">
+                    @Html.TextBoxFor(model => model.DonVi, new { @class = "form-control col-md-4 col-xs-12" })
+                    @Html.ValidationMessageFor(model => model.DonVi)
+                </div>
+            </div>
 
-                        <div class="form-group">
-                            <label class="control-label col-md-2 col-sm-2 col-xs-12" for="first-name">
-                                Phương pháp phân tích
-                            </label>
-                            <div class="col-md-4 col-sm-4 col-xs-12">
-                                <select class="form-control col-md-4 col-xs-12">
-                                    <option>Chọn phương pháp</option>
-                                    <option>Hóa nghiệm</option>
-                                    <option>Lý nghiệm</option>
-                                    <option>Sinh chất</option>
-                                    <option>Ống nghiệm</option>
-                                </select>
-                                @Html.ValidationMessageFor(model => model.PhuongPhapPhanTich)
-                            </div>
-                        </div>*@
+            <div class="form-group">
+                <label class="control-label col-md-2 col-sm-2 col-xs-12" for="first-name">
+                    Giá trị
+                </label>
+                <div class="col-md-4 col-sm-4 col-xs-12">
+                    @Html.TextBoxFor(model => model.GiaTri, new { @class = "form-control col-md-4 col-xs-12" })
+                    @Html.ValidationMessageFor(model => model.GiaTri)
+                </div>
+            </div>
 
-                        <h2 class="StepTitle">Thông tin kết quả</h2>
-                        <div class="row">
-                            <div class="col-md-2 col-sm-2 col-xs-2">
-                                <button id="btnThemMau" type="button"
-                                        data-toggle="modal" data-target="#themMauModal"
-                                        class="btn btn-success">
-                                    Thêm Kết Quả
-                                </button>
+            <div class="form-group">
+                <label class="control-label col-md-2 col-sm-2 col-xs-12" for="first-name">
+                    Phương pháp phân tích
+                </label>
+                <div class="col-md-4 col-sm-4 col-xs-12">
+                    <select class="form-control col-md-4 col-xs-12">
+                        <option>Chọn phương pháp</option>
+                        <option>Hóa nghiệm</option>
+                        <option>Lý nghiệm</option>
+                        <option>Sinh chất</option>
+                        <option>Ống nghiệm</option>
+                    </select>

```

Comment:
Em chưa address 1 thắc mắc khác của anh trên channel general slack
- Form KQ theo @htaphuong nói có 1 cột riêng là Phương Pháp Phân Tích (Khác Chỉ Tiêu Phân Tích). Trong From KQ của em. Anh nghĩ line này là nơi nhười chỉnh sửa FormKQ thực hiện việc chọn phương pháp phân tích? (Need verification)
- Cái này thì tùy vào từng chỉ tiêu mẫu mà hệ thống phải suggest các phương pháp phân tích khác nhau để người dùng chọn. Lưu ý theo data model ban đầu thì mỗi chỉ tiêu có nhiều phương pháp phân tích (one to many relation)
- Khi người dùng click dropdown chọn phương pháp phân tích. Hệ thống chỉ có thể suggest các phương pháp phân tích ứng với cái chỉ tiêu phân tích đã cho thôi?
- Xem thêm bàn luận chỉ tiêu phân tích và phương pháp phân tích của anh trên channel general slack